modules/mesh-task/variables.tf

```diff
@@ -165,7 +171,7 @@ variable "envoy_public_listener_port" {
   }
 
   validation {
-    error_message = "The envoy_public_listener_port must not conflict with the following ports that are reserved for Consul and Envoy: 8300, 8301, 8302, 8500, 8501, 8502, 8600, 19000."
+    error_message = "The envoy_public_listener_port must not conflict with the following ports that are reserved for Consul and Envoy: 8300, 8301, 8302, 8500, 8501, 8502, 8600, 10000, 19000."
```

Comment:
We can remove the validation for most of the Consul client agent ports here, I think: 8300, 8301, 8302, 8500, 8501, 8502. Without client agents

We should keep the DNS port reserved though (8600) because Consul Dataplane supports running a DNS server/proxy.